Formula/signal-cli.rb

```diff
@@ -0,0 +1,29 @@
+class SignalCli < Formula
+  desc "CLI and dbus interface for WhisperSystems/libsignal-service-java"
+  homepage "https://github.com/AsamK/signal-cli"
+  url "https://github.com/AsamK/signal-cli/releases/download/v0.6.1/signal-cli-0.6.1.tar.gz"
+  sha256 "550b389a25089ef2251d97997e06d57c70e9d4023684ff8682afef23ed2f1822"
+  depends_on :java => "1.7+"
+
+  def install
+    # pattern taken from activemq
+    libexec.install Dir["lib", "bin"]
+    (bin/"signal-cli").write_env_script libexec/"bin/signal-cli", Language::Java.java_home_env("1.6+")
+  end
+
+  test do
+    # test 1: checks class loading is working and version is correct
```

Comment:
If the second test also does the same thing then I don't see a need for the version check and lets stick with the more complex one.

Reply:
Fixed.